modules/apps/wiki/wiki-service/src/main/java/com/liferay/wiki/service/impl/WikiNodeLocalServiceImpl.java

```diff
@@ -550,6 +644,12 @@ protected void activate(
 			WikiPageDisplay.class.getName());
 	}
 
+	@Activate
```

Comment:
Please remove this method, since there is already an activate method in this class.